tests/smoke/rook_helper.go

```diff
@@ -0,0 +1,167 @@
+	rookOperatorCreatedTpr = "cluster.rook.io"
+)
+
+type RookHelper struct {
+	transportClient contracts.ITransportClient
+	isRookInstalled bool
+}
+
+func getPodSpecPath() string {
+	return filepath.Join(os.Getenv("GOPATH"), podSpecPath)
+}
+
+//method for create rook-operator via kubectl
+func (h *RookHelper) createK8sRookOperator(k8sHelper *utils.K8sHelper) error {
+
+	raw, err := ioutil.ReadFile(path.Join(getPodSpecPath(), rookOperatorFileName))
+
+	if err != nil {
+		return err
+	}
+	rookOperator := string(raw)
+
+	_, _, exitCode := h.transportClient.CreateWithStdin(rookOperator)
+
+	if exitCode != 0 {
+		return fmt.Errorf(string("Failed to create rook-operator pod; kubectl exit code = " + string(exitCode)))
+	}
+
+	fmt.Println()
+	if !k8sHelper.IsThirdPartyResourcePresent(rookOperatorCreatedTpr) {
+		return fmt.Errorf("Failed to start Rook Operator; k8s thirdpartyresource did not appear")
+	} else {
+		fmt.Println("Rook Operator started")
+	}
+
+	return nil
+}
+
+func (h *RookHelper) createK8sRookToolbox(k8sHelper *utils.K8sHelper) (err error) {
+
+	//Create rook toolbox
+	raw, err := ioutil.ReadFile(path.Join(getPodSpecPath(), rookToolsFileName))
+
+	if err != nil {
+		panic(err)
+	}
+
+	rookClient := string(raw)
+
+	_, _, exitCode := h.transportClient.CreateWithStdin(rookClient)
+
+	if exitCode != 0 {
+		return fmt.Errorf(string(exitCode))
+	}
+
+	if !k8sHelper.IsPodRunningInNamespace("rook-tools") {
+		return fmt.Errorf("Rook Toolbox couldn't start")
+	} else {
+		fmt.Println("Rook Toolbox started")
+	}
+
+	return nil
+}
+
+func (h *RookHelper) createk8sRookCluster(k8sHelper *utils.K8sHelper) error {
+
+	raw, err := ioutil.ReadFile(path.Join(getPodSpecPath(), rookClusterFileName))
+
+	if err != nil {
+		return err
+	}
+	rookCluster := string(raw)
+
+	_, _, exitCode := h.transportClient.CreateWithStdin(rookCluster)
+
+	if exitCode != 0 {
+		return fmt.Errorf("Failed to create rook-cluster pod; kubectl exit code = " + string(exitCode))
+	}
+
+	if !k8sHelper.IsServiceUpInNameSpace("rook-api") {
+		fmt.Println("Rook Cluster couldn't start")
+	} else {
+		fmt.Println("Rook Cluster started")
+	}
+
+	return nil
+}
+
+func (h *RookHelper) InstallRook() (err error) {
+	if h.isRookInstalled {
+		return
+	}
+
+	//Create rook operator
+	k8sHelp := utils.CreatK8sHelper()
+
+	err = h.createK8sRookOperator(k8sHelp)
+	if err != nil {
+		panic(err)
+	}
+
+	time.Sleep(10 * time.Second) ///TODO: add real check here
+
+	//Create rook cluster
+	err = h.createk8sRookCluster(k8sHelp)
+	if err != nil {
+		panic(err)
+	}
+
+	time.Sleep(5 * time.Second)
+
+	//Create rook client
+	err = h.createK8sRookToolbox(k8sHelp)
+	if err != nil {
+		panic(err)
+	}
+
+	h.isRookInstalled = true
+
+	return nil
+}
+
+func NewRookHelper() (*RookHelper, error) {
+
+	transportClient := transport.CreateNewk8sTransportClient()
+
+	return &RookHelper{
+		transportClient: transportClient,
+		isRookInstalled: false,
+	}, nil
+}
```

Comment:
We should move this under utils or common section so that other tests can re-use this.  
We should also add Uninstall rook Helper, so tests can clean up after runs.